yarn.lock

```diff
@@ -9,11 +9,23 @@
   dependencies:
     "@babel/highlight" "^7.12.13"
 
+"@babel/code-frame@^7.14.5":
```

Comment:
I would recommend running `yarn upgrade` to get rid of the duplicate packages.

Reply:
Done!